build.gradle.kts

```diff
@@ -127,5 +127,15 @@ publishing {
                 }
             }
         }
+        if ("true" == project.findProperty("publishGHP")) {
```

Comment:
Here I would check if the credentials are available and not use this additional parameter. There are already separate publish tasks for each repository so that users can choose where to publish.

Reply:
See https://github.com/modelix/modelix/blob/mps/2020.3/build.gradle#L110-L121